lib/BulkRenewalDialog/BulkRenewInfo.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import Button from '@folio/stripes-components/lib/Button';
+import Icon from '@folio/stripes-components/lib/Icon';
+import Layout from '@folio/stripes-components/lib/Layout';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
+
+import BulkRenewedLoansList from './BulkRenewedLoansList';
+
+class BulkRenewInfo extends React.Component {
+  static propTypes = {
+    stripes: PropTypes.object,
+    onCancel: PropTypes.func,
+    successRenewals: PropTypes.arrayOf(PropTypes.object),
+    failedRenewals: PropTypes.arrayOf(PropTypes.object),
+    loanPolicies: PropTypes.object,
+    requestCounts: PropTypes.object,
+    errorMessages: PropTypes.object,
+  }
+
+  constructor(props) {
+    super(props);
+    this.connectedLoanList = props.stripes.connect(BulkRenewedLoansList);
+  }
+
+  render() {
+    const { successRenewals, failedRenewals } = this.props;
+    const iconAlignStyle = { display: 'flex', alignItems: 'center' };
+
+    return (
+      <div>
+        <div style={iconAlignStyle}>
+          { failedRenewals.length > 0 ?
+            <span style={iconAlignStyle}>
+              <Icon size="medium" icon="validation-error" color="red" />
```

Comment:
Can you use the new `status` prop for `Icon` here instead of `color`? Technically, errors aren't in red but `#900` and I went ahead and did the work of selecting the correct colours from `variables.css` with that `status` prop.

The PR is folio-org/stripes-components#427. You'll probably have to bump your stripes-components dependency version.

Reply:
@doytch Ya, was following that PR. Will do the needful.